src/notebooks/execution/cellExecutionMessageHandler.ts

```diff
@@ -30,7 +30,6 @@ import { CellExecutionCreator } from './cellExecutionCreator';
 import { IApplicationShell } from '../../platform/common/application/types';
 import { disposeAllDisposables } from '../../platform/common/helpers';
 import { traceError, traceWarning } from '../../platform/logging';
-import { RefBool } from '../../platform/common/refBool';
```

Comment:
Don't need such a class, all we need is a property (added comments what the property does with reference to Jupyter docs)